### PR TITLE
Update release s2i

### PR DESCRIPTION
**Pull request by zmhassan**
# Description

Adding s2i script to build images and publish to quay with beta release tag.

### CODE REVIEW

File: scripts/build-s2i-push-quay.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
+VERSION="0.0.1b5"
```
[the remark this answers is not in the document]
Comment (zmhassan) [reply scripts/build-s2i-push-quay.sh]: Yes. I'll follow this convention.

**Comment by zmhassan** [comment]
@HumairAK @4n4nd Can you'll review this?

**Comment by zmhassan** [comment]
@goern Thank you I've added the UBI images.

File: README.rst [hunk 2]
```diff
@@ -27,12 +27,16 @@ Install Log Anomaly Detector (LAD):
 
 .. code-block:: shell
 
-   pip install git+https://github.com/AICoE/log-anomaly-detector.git
+   pip install log-anomaly-detector
 
 .. note::
 
    LAD requires python 3.6
```
Comment (4n4nd) [inline README.rst]: @zmhassan I think the formatting didn't work here but it worked in the readthedocs page

**Comment by zmhassan** [comment]
@4n4nd Please review and merge this ?

**Pull request merged**